app/models/appointment_slot_preset.rb

```diff
@@ -1,8 +1,29 @@
 class AppointmentSlotPreset < ActiveRecord::Base
+  include PushUpdatesMinimal
+
+  validate :no_conflict, on: :update
+
   validates :day, numericality: { only_integer: true, greater_than: 0, less_than: 8  }
   validates :hours, numericality: { only_integer: true, greater_than_or_equal_to: 0, less_than: 24 }
   validates :minutes, numericality: { only_integer: true, greater_than_or_equal_to: 0, less_than: 60 }
   validates :quota, numericality: { only_integer: true, greater_than_or_equal_to: 1 }
 
   scope :ascending, -> { order('hours ASC').order('minutes ASC') }
+
+  after_save :push_changes
+  after_destroy :push_changes
+  push_targets [ Channel::STOCK_CHANNEL ]
```

Comment:
I wonder if this could be ORDER_FULFILMENT_CHANNEL as it is only related to the order part of the stock app.